packages/react-component-event-listener/src/useEventListener.ts

```diff
@@ -17,25 +17,38 @@ const useEventListener = <T extends EventTypes>(options: EventListenerOptions<T>
     return latestListener.current(event)
   }, [])
 
+  if (process.env.NODE_ENV !== 'production') {
+    React.useEffect(() => {
```

Comment:
We have a check for both being set but no propTypes/check for none. In that case you will receive a cryptic error

Reply:
Add a check 👍